### PR TITLE
move app.js to i18n

### LINKED ISSUE #79: Respect the URL parameter specifying a language
currently, the app doesn't adjust the language according to the URL parameter specifying it. For example: if we go to the following URL, the app will be displayed in english although the URL is specifying Hebrew:
https://sensoryinterface.com/?lang=he#

### LINKED ISSUE #80: Move public/js/app.js functionality into public/languages/i18n.js
Try to avoid having to specify particular elements such as `aboutP1`, `aboutP2` etc.

### PR DESCRIPTION
**Pull request by BitanTomer**
Fixes #80
Fixes #79

### CODE REVIEW

File: public/languages/i18n.js [hunk 1]
```diff
@@ -58,3 +58,30 @@ const initLanguage = () => {
...
   setLanguage(lang);
 };
+
+//Save on the design between language
+const changeLanguage = (language) => {
+  $('#language-popup').modal('toggle');
+  console.log(language);
+  setLanguage(language);
+  if (lang === 'he') {
+    document.body.setAttribute('dir', 'rtl');
```
Comment (oferb) [inline public/languages/i18n.js:68]: Let's try to do this without specifying element IDs and languages.

Comment (BitanTomer) [reply public/languages/i18n.js]: I think I fixed it,  it is ok now?

Comment (oferb) [reply public/languages/i18n.js]: @mohammad-suliman can you deploy to dev and we check it out?

File: public/index.html [hunk 2]
```diff
@@ -351,13 +356,16 @@ <h5 class="modal-title w-100" id="select-language-header">Change language</h5>
...
-  <script defer src="languages/he.js"></script>
-  <script defer src="languages/en.js"></script>
-  <script defer src="./js/utils.js"></script>
-  <script defer src="./js/app.js"></script>
+  <script src="./js/utils.js"></script>
+  <script src="languages/i18n.js"></script>
+  <script src="languages/he.js"></script>
+  <script src="languages/en.js"></script>
+  <!-- initialize translation -->
+  <script>
```
Comment (mohammad-suliman) [inline public/index.html:364]: @oferb  What do you think of this? I did it that way BTW 😊

**Pull request merged**